<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-15" />
<meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="Author" content="A.T. Garcia-Sosa, Alfonso Tlatoani Garcia Sosa, Alfonso T. Garcia-Sosa, pdf">
   <meta name="description=" contents="Research description, Dr. Alfonso T. Garcia-Sosa, PhD., A.T. Garcia-Sosa, pdf, machine learning, big data, data mining, 1986,1988,1999,1ppk,2000,2003,2004,2005,2006,2008,2009,2010,2012,2013,2014,2a6,2c9,3a4,3app,Ab,ability,Abl,able,above,abs,absolute,Abstract,accessible,according,account,accuracy,accurate,achieved,Achieving,acids,across,acs,active,actives,activity,adapted,addition,additional,address,adiabatic,adjacent,administered,administration,adopt,ADP,advances,affects,affinities,affinity,after,against,agonists,agreement,Aid,aided,aim,Alberts,alchemical,Alfonso,alfx,algorithms,align,alignment,all,allow,allowed,allowing,allows,also,alt,always,amide,amino,among,amount,an,analyses,analysis,analytics,analyzed,and,ang,angles,Animation,Animations,antagonists,anti,antidote,antitarget,Antitargets,apo,appearing,appears,application,applications,applied,approach,approaching,approx,approximation,aqueous,are,area,arial,Aring,arises,aromatic,arrangements,Article,as,ASAP,asp,assess,assign,Assigning,associated,assume,at,atom,atomic,atoms,attempts,au,Author,authorPDF,available,average,averages,avian,b,b4,B88,balance,based,basis,battery,be,because,Becke,been,before,being,below,beneficial,benefits,Besides,best,beta,better,between,Bfactor,bias,bin,bind,binders,binding,bioavailability,Biochemistry,biological,biology,BioMed,biomolecular,Bioorg,Bioorganic,blood,blue,bmc,body,bond,bonded,bonds,BoNTA,BoNTA_inh,bootstrapping,BORDER,both,Botulinum,bound,boundary,br,bridge,bridges,bridging,browser,built,bulk,but,button,BW,by,C,calcer,calcIt,Calculate,calculated,calculating,calculation,Calculations,calculator,Calculators,calibration,called,can,cancer,candidate,candidates,carbons,carried,case,cases,Castro,catalytic,catch,categories,category,cationic,cause,CDK2,cell,cells,center,central,centres,certain,cgi,chains,challenges,change,changes,changing,characteristics,characterization,characterize,characterized,charged,charset,Chem,Chemical,chemically,Chemistry,ry,chlorthalidone,chronic,ci049694b,ci200203h,ci200587h,ci3005786,ci500300u,ci800242z,cite,Clark,clashes,clashing,class,classification,close,CMC,CML,CO,code,co
ed,Coeffa,Coeffb1,Coeffb2,Coeffb3,coefficent,coefficient,coefficients,cognate,co
llection,com,combination,combinations,Combinatorial,combine,combined,common,comm
only,compare,compared,comparisons,compete,completely,complex,complexation,comple
xed,complexes,component,components,composing,compound,compounds,comprehensive,Co
mput,Computational,computationally,compute,computed,computer,concentration,concl
uded,concurrent,conditions,conducted,confidence,configuration,confirmed,conforma
tion,conformational,conformations,consensus,Consequences,conserves,consider,cons
ideration,considerations,considered,considering,consistent,consists,constraints,
Contact,contacts,containing,Content,contents,continuous,contribute,coordinate,co
ordination,copy,copyright,core,correcting,correlated,correlation,correlations,co
uld,CPP,CPPs,CRETRY,critical,cross,crstal,crystal,crystallographic,crystallograp
hically,Csaba,css,Curr,Current,Currently,cyan,cycles,Cyclin,CYP,CYPs,cytochrome,
d,data,databank,dataset,date,de,dealt,Dean,deeper,define,deg,delicate,deliver,De
lta,demonstrate,density,depend,dependent,deployed,derivation,derive,Derived,desc
ribe,described,describing,description,descriptor,descriptors,Design,designed,des
ired,desolvation,determinations,determine,determined,determining,developability,
developed,development,DFT,DGauss,did,difference,different,differentiation,differ
ing,digital,dioxygen,direct,directed,discover,discovery,discriminate,Discriminat
ion,discussed,Disease,diseases,displace,Displaceable,displaced,displacement,diss
ociated,distance,distances,distinction,distinguish,distinguished,Distinguishing,
Distributed,distribution,div,Diverse,diversity,divided,dmmsclick,do,docked,docking,
DOCTYPE,document,does,DOI,domain,done,download,downregulation,Dr,driving,Drug
,DrugLogit,drugs,DTD,due,dummy,during,dx,dynamic,dynamical,dynamics,e,EA,eacute,
early,easily,ee,effect,effective,effectively,Effects,efficacy,efficiencies,effic
iency,effluxing,EI,either,electron,electronic,elucidated,em,embedded,emphasizing
,employed,EN,enabled,Endopeptidase,energetic,energetics,energies,energy,engines,
enhancing,Enthalpic,enthalpically,enthalpy,Entropic,entropically,environment,enz
yme,enzymes,equally,equations,equiv,err,especially,establish,estilo,estimation,E
T,eta,ethyl,eV,evaluate,evaluation,events,example,examples,exchange,Exemplificat
ion,exists,exp,experiment,experimental,experimentally,explain,explicit,explored,
exposed,extended,extensively,external,extracting,f,F4,F4abc,F5,face,faced,factor
,factors,family,fast,favor,favorably,favourable,Fe,features,FeO,FeO2,feo2gtex,fe
w,field,fields,Fig1_cpps,figforwebpcs4,FigH5N1jcim,Figjcc_ei,figstates,figure,Fi
gure1,Fig_triangular,filtering,filters,findings,first,Firth,fit,five,fluctuation
,focused,fold,following,font,for,foreseen,form,formally,formation,found,four,fra
mework,frecuency,Free,freely,frequencies,frequently,from,full,fulltext,Fully,fun
ction,functional,functionals,functions,fundamental,further,future,g,ga,gain,gaJs
Host,gal_cdk2,Garamond,Garc,Garcia,GarciaSosaAT2010jcc,general,generalized,gener
ated,generating,Generation,genre,geometry,GGA,gif,given,good,google,gradient,gra
dual,Grid,Ground,group,groups,growing,GS,guide,h,h2,H274Y,h3,H5N1,had,hampered,h
armonic,has,have,having,head,heavy,help,hermes,Het,Hetenyi,Hig
h,highlights,highly,hit,hits,HIV,HIVrt,Hot,how,However,hr,href,HSV,htm
l,HTMLSTART,http,https,hybridization,hydrated,hydration,hydrogen,hydrophilicity,
hydroxyl,hydwat,I,iacute,ID,identification,identified,identifier,identify,
impact,implemented,implicated,implications,importance,important,impo
sed,improve,improved,Improvement,improves,Improving,in,include,included,includes
,Including,Inclusion,incorporating,increased,increasing,Indeed,independent,indep
endently,index,indicates,indications,indices,indirect,individual,Inf,infin,influ
ence,influencing,influenza,Informatics,information,inhibit,inhibition,Inhibitor,
inhibitors,initio,innovative,input,inputA,inputB,inputC,inside,insight,instead,I
ntegrase,integration,interact,interaction,interactions,interest,interested,inter
esting,interfere,intermolecular,International,interpretable,Interpretation,inter
science,intervals,intestines,into,introduce,introduced,intuitive,involved,involv
es,involving,in_manuscript,ion,ionization,IP,Iron,is,ISO,Iss,Issue,it,its,J,java
script,jcam,Journal,journals,jpg,js,justified,K,kcal,Kd,keeping,key,Ki,kinase,kn
owledge,known,L,lack,lactam,lang,Langel,larger,lead,leads,least,leeway,less,leuk
emia,level,levels,libraries,library,Ligand,ligands,likeness,limited,line,link,li
nks,lipophilic,lipophilicity,literature,liver,Lloyd,loadings,local,location,log,
logE,Logistic,long,loosely,low,lower,lowest,LSDA,luciferase,m,made,magenta,magni
tude,mainly,majority,makes,making,Makiyi,Mancera,many,mapped,Maran,Mare,matched,
Math,may,MD,means,measure,measures,Med,Medicinal,meta,metabolic,metabolism,metab
olizing,metal,metalloproteins,metapress,method,methods,methyl,micro,micromolar,M
illard,mimicking,minf,minimization,minus,MMDS,mode,Model,modeled,Modeling,Models
,moderate,modes,modification,modifications,modified,modifies,modify,modulate,moi
eties,Mol,molecular,molecule,molecules,more,most,movie,Mulliken,Multi,multibindi
ng,multiple,Multivariate,mutant,mutants,mutational,Mutations,my,myelogenous,myel
oid,N,NAD,name,Named,namely,NaN,natural,nature,nbsp,ndash,nearby,neccessary,
negative,neglect,neglected,neglecting,nervous,neuraminidase,neurotoxin,new,no,N
oC,non,nondrug,Nondrugs,nonhydrated,nontightly,normalize,not,Note,Novel,novo,NPA
C,ns,nucleic,number,numbers,Nusair,nyi,O,object,objective,observe,observed,obser
ving,obtained,occupied,occupy,octanol,of,OFeO,off,often,Oja,on,onClick,one,onlin
elibrary,only,open,openurl,optimal,optimization,optimized,or,oral,orally,orbital
,order,ordered,org,organ,original,Oseltamivir,other,others,out,outcome,outcomes,
output,over,overall,oxazepam,Oxygen,p,P450,P86,page,pages,pageTracker,pairs,Pang
,paper,parentheses,Park,PARP,parseFloat,part,particular,partition,partners,passi
ve,pathophysiology,patterns,PCA,pdb,pdf,pea1,pea10,pea11,pea12,pea14,pea15,pea16
,pea17,pea2,pea3,pea4,pea5,pea6,pea7,pea8,pea9,penetrating,penicillopepsin,pepti
de,peptides,per,Perdew,performance,performed,periodic,permutation,persists,persp
ective,perturbation,pharmaceutical,pharmacodynamic,pharmacokinetic,pharmacokinet
ics,pharmacological,pharmacology,pharmacophore,pharmacophores,pharmacophoric,pha
rmforwat,PhD,phenomena,phenyl,physically,physicochemical,physicochemically,plana
r,please,plot,png,point,points,poly,polymerase,polypharmacology,polysaccharide,p
opulation,poses,positive,possess,possibility,possible,postdocking,potency,potent
,potentials,pow,power,practical,pre,precision,predict,predicted,predicting,predi
ction,Predictions,predictive,predictors,prefiltering,pregnane,preOut1,preOut2,pr
eOut3,presence,present,previously,principal,prior,Probabilities,probability,prob
es,problems,process,processes,produced,products,profile,profiles,profiling,progr
am,programs,projection,projects,properties,property,protease,protein,proteins,pr
otocol,proves,provided,provides,providing,PUBLIC,publication,publications,PubMed
,pubs,pubs3,PXR,QSAR,QUA4,quantitative,Quantum,Quick,quinoline,R,
ranges,rank,ranked,ranking,ranks,rationalized,ratios,ray,re,read,readily,realistic,Recent,recently,receptor,receptro,recognition,recovery,reducing,regions,Regression,regressions,rel,relate,related,relationship,relative,relevance,relevant,
remain,remaining,removal,removed,removes,replace,replaced,replacement,replacing,
report,reported,reportedly,reports,repurposing,required,rerank,reranking,resampl
ing,Research,residue,residues,resistant,resolution,respective,resulted,results,r
euse,rev,rev2,reveal,reverse,reviewed,ribose,rigorous,ring,robustness,role,rough
,route,routes,RT,runs,s,s00894,safety,same,sampling,satisfactorily,Scaffold,scaf
folds,scenarios,SCF,Schmidt,Science,sciences,score,scored,scores,scoring,screen,
screening,script,SCSA,See,seen,select,selective,selectivity,self,sensitivity,sep
aration,sequentially,series,serotype,service,set,sets,settings,several,SH3,shape
,shell,should,show,showed,shown,shows,side,sidechains,sign,significance,signific
ant,Sild,silico,Sill,similar,Simple,sims,simulation,simulations,simultaneous,sim
ultaneously,since,Singly,siRNA,site,sites,size,small,smaller,so,solvation,Solven
t,some,sometimes,Sosa,SosaAT08,SosaAT09QSAR,SosaATcurrmedchem2012forweb,SosaATmo
linf2012PCA,SosaAT_molinf2010,SosaAT_sqer2013,sp,sp3,space,spatial,special,speci
es,specific,specificity,sphere,spheres,spin,split,Springer,springerlink,SQER,SRC
,SRETRY,ssl,stability,stable,stages,START,State,states,statistical,statistically
,stearyl,step,sterically,sticks,strategies,strategy,Strict,strong,stronger,stron
gly,structural,structure,structures,studied,studies,study,studying,style,stylesh
eet,sub,subsequent,substances,substitution,Substitutions,substrate,succeed,succe
ssive,such,suggest,suited,sulfotransferase,SULT,sup,superimposed,superposition,S
upporting,supporting_information,surface,survey,synthetic,system,systematic,
syst
ems,T,TABLE,tableparpfig,tailored,taking,Takkis,Tamiflu,tandfonline,target,targe
ted,targeting,targets,TD,techniques,Technology,temperature,termination,terms,tes
t,Tetrahedral,TEX,text,than,that,the,their,them,themodynamic,theory,there,Theref
ore,thermodynamic,thermodynamics,these,They,thick,thin,this,those,though,three,t
hreshold,thresholds,through,throughout,throughput,thus,thymidine,Tight,tightly,t
itle,Tlatoani,to,TOCdruglogit,TOC_watjcim2013,Todorov,together,top,topological,t
otal,toxic,toxicity,toxin,TR,training,Transcriptase,transcriptases,transition,tr
iaging,Triangular,triazine,try,Tulp,tumor,turn,two,txt,Type,typo,tyrosine,U,UA,u
ndergraduate,underlying,understand,unescape,unfavourable,UniChem,unique,upon,use
,used,useful,usefulness,using,ut,Uuml,v,v4c,valid,validated,valuable,value,value
s,var,variable,variables,Variant,variety,Verlag,vertical,very,vibration,vibratio
nal,videos,view,virtual,virus,visual,visualized,Vol,Vosko,VWN,W,w3,W3C,was,water
,waterphore,waters,WaterScore,way,We,web,webFeO2aa,webfigcdk2wat,webfigFeO2bb,weight,well,were,when,where,which,while,white,Wiener,wild,wiley,Wilk,will,with,wit
hin,without,work,would,write,www,www3,X,XHTML,xhtml1,xml,xmlns,Y,Y181C,yellow,Yo
u,your,zero,zinc,Zn,"/>

<title>Research: Alfonso T. Garcia-Sosa (A.T. Garcia-Sosa). Computational Chemistry and Drug Design. PDF</title>
<link type="text/css" rel="stylesheet" href="estilo.css"  />

<script TYPE="text/javascript">
<!--
function calcIt() {
      var Bfactor = parseFloat(document.calcer.inputA.value)
      var SCSA = parseFloat(document.calcer.inputB.value)
      var NPAC = parseFloat(document.calcer.inputC.value)
      var Coeffa = 76.442
      var Coeffb1 = 5.278
      var Coeffb2 = 2.166
      var Coeffb3 = 84.458
      var preOut1 = Coeffa - (Coeffb1*Bfactor) - (Coeffb2*SCSA) + (Coeffb3*NPAC)
      var logE = 0.434294481
      var preOut2 = preOut1*logE
      var preOut3= Math.pow(preOut1, preOut2)
      document.calcer.output.value = preOut3/(1+preOut3)

      }
  // -->
</script>

</head>
<body>
<font face="Garamond">

<h2>Research</h2>
<hr>
<br>
</font>

<p class="research">
<div style="font-family:arial;font-size:110%;line-height:150%">
I'm interested in the fields of drug design, biomolecular simulation, and computational chemistry, among others.
<br><br>
Quick links:<br>
<TABLE BORDER="1">
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea23">Machine learning in drug discovery and design</a>  &nbsp;&nbsp;
</TD>
<TD>
&nbsp;&nbsp; <a href="research.html#pea20">Inhibitors against Leishmaniasis</a>  &nbsp;&nbsp;
</TD>
</TR>
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea24">Cobalt-Ligand complexes inhibit bacterial film and quorum sensing</a>  &nbsp;&nbsp;
</TD>

<TD>
&nbsp;&nbsp; <a href="research.html#pea19">Small, low-toxicity, novel compounds that inhibit HIV-1</a>  &nbsp;&nbsp;
</TD>
</TR>
<!--- <TD>
&nbsp;&nbsp; <a href="research.html#pea18">VS for HIV-1 Protease</a>  &nbsp;&nbsp;
</TD>
<TD>
&nbsp;&nbsp; <a href="research.html#pea17">Triangular numbers in re-ranking for HIV-1 Integrase</a>  &nbsp;&nbsp;
</TD> -->
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea16">siRNA binding to cell-penetrating peptides</a>  &nbsp;&nbsp;
</TD>
<TD>
&nbsp;&nbsp; <a href="research.html#pea15">Ligand hydration and binding in 2,332 crystal structures</a>  &nbsp;&nbsp;
</TD>
</TR>
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea14">DrugLogit <sup>&copy;</sup> and DrogLogit online calculators</a> &nbsp;&nbsp;
</TD>
<!--- <TD>
&nbsp;&nbsp; <a href="research.html#pea12">PCA and drug ratios</a> &nbsp;&nbsp;
</TD>
</TR>
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea11">Antitargets and virtual screening hits for HIV-1 RT</a>
&nbsp;&nbsp;
</TD> --->
<TD>
&nbsp;&nbsp; <a href="research.html#pea10">Free energy modification of ligand functional groups using waters</a>
&nbsp;&nbsp;
</TD>
</TR>
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea9">Ligand and drug efficiency indices</a> &nbsp;&nbsp;
</TD>
<TD>
&nbsp;&nbsp; <a href="research.html#pea8">Inhibitor design for H5N1 influenza</a> &nbsp;&nbsp;
</TD>
</TR>
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea7">Botulinum toxin inhibitors by MD sims of Zn metalloproteins</a> &nbsp;&nbsp;
</TD>
<TD>
&nbsp;&nbsp; <a href="research.html#pea6">WaterScore and online calculator </a> &nbsp;&nbsp;
</TD>
</TR>
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea5">WaterScore calculator <sup>&copy;</sup></a> &nbsp;&nbsp;
</TD>
<TD>
&nbsp;&nbsp; <a href="research.html#pea4">Tight waters in pharmacophores</a>
&nbsp;&nbsp;
</TD>
</TR>
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea3">PARP inhibitor design</a>
&nbsp;&nbsp;
</TD>
<TD>
&nbsp;&nbsp; <a href="research.html#pea2">CDK2 inhibitor design</a>
&nbsp;&nbsp;
</TD>
</TR>
<TR>
<TD>
&nbsp;&nbsp; <a href="research.html#pea1">FeO<sub>2</sub>, FeO<sub>2</sub><sup>+</sup>, FeO<sub>2</sub><sup>-</sup>, and DFT</a> &nbsp;&nbsp;
</TD>
<TD>
&nbsp;&nbsp; <a href="publications.html">Publications</a>&nbsp;&nbsp;
</TD>
</TR>
<TR>
<TD>
&nbsp;&nbsp; <a href="videos.html">Animations</a> &nbsp;&nbsp;
</TD>
<TD>
&nbsp;&nbsp; <a href="index.html">Contact</a> &nbsp;&nbsp;
</TD>
</TR>
</TABLE>

</div></p>
<br>
<hr>
<br>
<div style="font-family:arial;font-size:110%;line-height:150%;">
<div style="font-family:arial;font-size:110%;line-height:150%;">
	<hr>
	<br>
	<a name="pea23"><h3>Machine learning in drug design and discovery</h3></a>
	<center>
		<br>

		<img SRC="WIREs.png" width="65%" border="0" alt="Machine learning (ML) is becoming capable of transforming biomolecular interaction description and calculation, promising an impact on molecular and drug design, chemical biology, toxicology, among others. The first improvements can be seen from biomolecule structure prediction to chemical synthesis, molecular generation, mechanism of action elucidation, inverse design, polypharmacology, organ or issue targeting of compounds, property and multiobjective optimization. Chemical design proposals from an algorithm may be inventive and feasible. Challenges remain, with the availability, diversity, and quality of data being critical for developing useful ML models; marginal improvement seen in some cases, as well as in the interpretability, validation, and reuse of models. The ultimate aim of ML should be to facilitate options for the scientist to propose and undertake ideas and for these to proceed faster. Applications are ripe for transformative results in understudied, neglected, and rare diseases, where new data and therapies are strongly required. Progress and outlook on these themes are provided in this study">

		<br><br>
<img SRC="Benfordslaw.png" width="35%" border="0" alt="Aim: The explosion of data based technology has accelerated pattern mining. However, it is clear that quality and bias of data impacts all machine learning and modeling. Results & methodology: A technique is presented for using the distribution of first significant digits of medicinal chemistry features: logP, logS, and pKa. experimental and predicted, to assess their following of Benford's law as seen in many natural phenomena. Conclusion: Quality of data depends on the dataset sizes, diversity, and magnitudes. Profiling based on drugs may be too small or narrow; using larger sets of experimentally determined or predicted values recovers the distribution seen in other natural phenomena. This technique may be used to improve profiling, machine learning, large dataset assessment and other data based methods for better (automated) data generation and designing compounds. Machine learning and other technology depends critically on quality of data Benford's law can indicate data follows natural phenomena easy, fast, statistical Drug design impacted by FSD of experiment and predicted logP, pKa, solubility distributions Method suited for large datasets Keywords bias chemical library data science distribution drug design drug discovery drug likeness filters machine learning">
<br>
<img SRC="yosipofFig2.png" width="50%" border="0" alt="Data mining approaches can uncover underlying patterns in chemical and pharmacological property space decisive for drug discovery and development. Two of the most common approaches are visualization and machine learning methods. Visualization methods use dimensionality reduction techniques in order to reduce multi-dimension data into 2D or 3D representations with a minimal loss of information. Machine learning attempts to find correlations between specific activities or classifications for a set of compounds and their features by means of recurring mathematical models. Both models take advantage of the different and deep relationships that can exist between features of compounds, and helpfully provide classification of compounds based on such features or in case of visualization methods uncover underlying patterns in the feature space. Drug-likeness has been studied from several viewpoints, but here we provide the first implementation in chemoinformatics of the t-Distributed Stochastic Neighbor Embedding t-SNE method for the visualization and the representation of chemical space, and the use of different machine learning methods separately and together to form a new ensemble learning method called AL Boost. The models obtained from AL Boost synergistically combine decision tree, random forests RF, support vector machine SVM, artificial neural network ANN, k nearest neighbors kNN, and logistic regression models. In this work, we show that together they form a predictive model that not only improves the predictive force but also decreases bias. This resulted in a corrected classification rate of over 0.81, as well as higher sensitivity and specificity rates for the models. In addition, separation and good models were also achieved for disease categories such as antineoplastic compounds and nervous system diseases, among others. Such models can be used to guide decision on the feature landscape of compounds and their likeness to either drugs or other characteristics, such as specific or multiple disease-categoryies or organs of action of a molecule.Keywords machine-learning, drug, data-mining, logistic, organ, drug design, multi-target">
<br>
	</center>

	<br><br>
	Pe&ntilde;a-Guerrero J., Nguewa P.,* <b>Garc&iacute;a-Sosa A. T.*</b> "Machine Learning, Artificial Intelligence, and Data Science Breaking into Drug Design and Neglected Diseases" <i>WIREs Computational Molecular Science</i> <b>2021</b>, e1513, <i>Published online 5th January 2021</i>, <a href="https://wires.wiley.com/WileyCDA/WiresArticle/wisId-WCMS1513.html">[HTML]</a>, <a href="https://doi.org/10.1002/wcms.1513">[DOI]</a>, <a href="WIREs.pdf"> [PDF]</a>
	<br>
	<br><b>Garc&iacute;a-Sosa A. T.*</b> "Androgen Receptor Binding Category Prediction with Deep Neural Networks and Structure-, Ligand-, and Statistically-Based Features" <i>Molecules</i> <b>2021</b>, 26:1285 <a href="https://doi.org/10.3390/molecules26051285">[DOI]</a>, <a href="https://www.mdpi.com/1420-3049/26/5/1285/pdf">[PDF]</a><br>

	<br><b>Garc&iacute;a-Sosa A. T.*</b> "Benford's Law in Medicinal Chemistry: Implications for Drug Design", <i>Future Medicinal Chemistry</i>, 2019, Vol. 11, Iss. 17, 2247-2253, <a href="https://doi.org/10.4155/fmc-2019-0006">[DOI]</a>, <a href="https://doi.org/10.4155/fmc-2019-0006">[HTML]</a>, <a href="fmc-2019-0006latest.pdf">[PDF]</a>, <a href="download.html">[Data]</a>
<br>
<br>Yosipof A., Guedes R. C., <b>Garc&iacute;a-Sosa A. T.</b>* "Data Mining and Machine Learning Models for Predicting Drug Likeness and their Disease or Organ Category", <i>Frontiers in Chemistry</i>, 2018, Vol. 6, 162, <a href="https://doi.org/10.3389/fchem.2018.00162"> [DOI]</a>, <a href="https://www.frontiersin.org/articles/10.3389/fchem.2018.00162/pdf"> [PDF]</a>
<b><a href="https://www.technologynetworks.com/drug-discovery/articles/data-visualization-in-biopharma-leveraging-ai-vr-and-mr-to-support-drug-discovery-320108?fbclid=IwAR2mdhtKtxrObLLG3GqsLouHN3dwwGK_tvHcukvqakm5FNJYZBeWaO3q1Y8">Highlighted in the Specialty News. Technology Networks "Data Visualization in Biopharma: Leveraging AI, VR, and MR to Support Drug Discovery"</a></b>

<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>

	<hr>
	<a name="pea20">
<h3>Inhibitors of <i>Leishmania</i></h3>
<center>
<img SRC="ijms-22-10493-g011.png" width="100%" border="0" alt="Since many of the currently available antileishmanial treatments exhibit toxicity, low effectiveness, and resistance, search and validation of new therapeutic targets allowing the development of innovative drugs have become a worldwide priority. This work presents a structure-based drug discovery strategy to validate the Lmj_04_BRCT domain as a novel therapeutic target in Leishmania spp. The structure of this domain was explored using homology modeling, virtual screening, and molecular dynamics studies. Candidate compounds were validated in vitro using promastigotes of Leishmania major, L. amazonensis, and L. infantum, as well as primary mouse macrophages infected with L. major. The novel inhibitor CPE2 emerged as the most active of a group of compounds against Leishmania, being able to significantly reduce the viability of promastigotes. CPE2 was also active against the intracellular forms of the parasites and significantly reduced parasite burden in murine macrophages without exhibiting toxicity in host cells. Furthermore, L. major promastigotes treated with CPE2 showed significant lower expression levels of several genes (α-tubulin, Cyclin CYCA, and Yip1) related to proliferation and treatment resistance. Our in silico and in vitro studies suggest that the Lmj_04_BRCT domain and its here disclosed inhibitors are new potential therapeutic options against leishmaniasis">
<br><br>
<img SRC="GraphicalAbstractwBackgrnd.png" width="25%" border="0" alt="Due to the lack of approved vaccines against human leishmaniasis and the limitations of the current chemotherapy inducing side effects and drug resistance, development of new, effective chemotherapeutic agents is essential. This study describes the synthesis of a series of novel oxadiazoles and indolizine-containing compounds. The compounds were screened in silico using an EIIP/AQVN filter followed by ligand-based virtual screening and molecular docking to parasite arginase. Top hits were further screened versus human arginase and finally against an anti-target battery to tag their possible interactions with proteins essential for the metabolism and clearance of many substances. Eight candidate compounds were selected for further experimental testing. The results show measurable in vitro anti-leishmanial activity for three compounds. One compound with an IC50 value of 2.18 ÂµM on Leishmania donovani intramacrophage amastigotes is clearly better positioned than the others as an interesting molecular template for further development of new anti-leishmanial agents">
</center>

<br>Pe&ntilde;a-Guerrero J., Fern&aacute;ndez-Rubio C., Burguete-Mikeo A
., El-Dirany R., <b>Garc&iacute;a-Sosa A. T.*</b>, Nguewa P.* "Discovery and Val
idation of Lmj_04_BRCT Domain, a Novel Therapeutic Target: Identification of Can
didate Drugs for Leishmaniasis" <i>International Journal of Molecular Sciences</i> <b>2021</b>, 22:10493 <a href="https://doi.org/10.3390/ijms221910493">[DOI]</
a>, <a href="https://www.mdpi.com/1422-0067/22/19/10493/pdf">[PDF]</a> 
	        <br>

<br>Stevanovic S., Sencanski M., Danel M., Menendez C., Belguedj R., Bouraiou A., Nikolic K., Cojean S., Loiseau P. M., Glisic S., Baltas M.,<b> Garc&iacute;a-Sosa, A. T.*</b> "Synthesis, <i>In Silico</i>, and <i>In Vitro</i> Evaluation of Antileishmanial Activity of Oxadiazoles and Indolizine_Containing Compounds Flagged Against Antitargets", <i>Molecules</i>, 2019, Vol. 24, Iss. 7, 1282, <a href="https://doi.org/10.3390/molecules24071282">[DOI]</a>, <a href="https://www.mdpi.com/1420-3049/24/7/1282/htm">[HTML]</a>, <a href="https://www.mdpi.com/1420-3049/24/7/1282/pdf">[PDF]</a><br>
<br><b>Garc&iacute;a-Sosa A. T.</b>* "Designing Ligands for <i>Leishmania</i>, <i>Plasmodium</i>, and <i>Aspergillus</i> N-Myristoyl Transferase with Specificity and Antitarget Safe Virtual Libraries", <i>Current Computer-Aided Drug Design</i>, 2018, Vol. 14, No. 2, 131-141, <a href="http://dx.doi.org/10.2174/1573409914666180308163231">[DOI]</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/29521245"> [PubMed]</a>
<br>
<br>Glisic S., Sencanski M., Perovic V., Stevanovic S., <b>Garc&iacute;a-Sosa A.
 T.*</b> "Arginase Flavonoid Antileishmanial <i>In Silico</i> Inhibitors Flagged
 Against Antitargets" <i>Molecules</i> <b>2016</b>, 21(5):589 <a href="http://ww
w.mdpi.com/1420-3049/21/5/589">[HTML]</a>, <a href="http://www.mdpi.com/1420-304
9/21/5/589/pdf">[DOI]</a>, <a href="http://www.mdpi.com/1420-3049/21/5/589/pdf">
[PDF]</a>
<br>

<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.<br></p>


<hr>
<a name="pea24">
<h3>Cobalt-Ligand complexes inhibit bacterial film and quorum sensing</h3>
<center>
<img SRC="molecules-23-01385-sch001.png" width="50%" border="0" alt="Pseudomonas aeruginosa is one of the most dreaded human pathogens, because of its intrinsic resistance to a number of commonly used antibiotics and ability to form sessile communities (biofilms). Innovative treatment strategies are required and that can rely on the attenuation of the pathogenicity and virulence traits. The interruption of the mechanisms of intercellular communication in bacteria (quorum sensing) is one of such promising strategies. A cobalt coordination compound (Co(HL)2) synthesized from (E)-2-(2-(pyridin-2-ylmethylene)hydrazinyl)-4-(p-tolyl)thiazole (HL) is reported herein for the first time to inhibit P. aeruginosa 3-oxo-C12-HSL-dependent QS system (LasI/LasR system) and underling phenotypes (biofilm formation and virulence factors). Its interactions with a possible target, the transcriptional activator protein complex LasR-3-oxo-C12-HSL, was studied by molecular modeling with the coordination compound ligand having stronger predicted interactions than those of co-crystallized ligand 3-oxo-C12-HSL, as well as known-binder furvina. Transition metal group 9 coordination compounds may be explored in antipathogenic/antibacterial drug design.
Keywords: antibacterial resistance; antivirulence/antipathogenic compounds; biofilm prevention; cobalt complex; furvina; pyocyanin; pyoverdine; quorum sensing inhibition; transcriptional activator protein LasR"/>
</center>
<br>
<br>Borges A., Sim&otilde;es M., Todorovic T. R., Filipovic, N. R., <b>Garc&iacute;a-Sosa A. T.</b>* "Cobalt Complex with Thiazole_Based Ligand as New <i>Pseudomonas aeruginosa</i> Quorum Quencher, Biofilm Inhibitor, and Virulence Attenuator", <i>Molecules</i>, 2018, Vol. 23, 1385, <a href="https://dx.doi.org/10.3390/molecules23061385">[DOI]</a>, <a href="http://www.mdpi.com/1420-3049/23/6/1385/htm">[HTML]</a>, <a href="http://www.mdpi.com/1420-3049/23/6/1385/pdf">[PDF]</a>
<br>

<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>

<hr>
<a name="pea21"><h3>Delivering DNA in vivo with cell penetrating peptides</h3></a>

									<b>Selected for Cover Figure and Editorial Comment</b>
									<center>
										<img SRC="jcr_cover_figure.jpg" border="0" alt="As the field of gene therapy progresses, an increasingly urgent need has arisen for efficient and non-toxic vectors for the in vivo delivery of nucleic acids. Cell-penetrating peptides (CPP) are very efficient transfection reagents in vitro, however, their application in vivo needs improvement.
										To enhance in vivo transfection we designed various CPPs based on previous knowledge of internalization studies and physiochemical properties of NickFect (NF) nanoparticles. We show that increment of the helicity of these Transportan10 analogues improves the transfection efficiency. We rationally design by modifying the net charge and the helicity of the CPP a novel amphipathic Î±-helical peptide NF55 for in vivo application. NF55 condenses DNA into stable nanoparticles that are resistant to protease degradation, promotes endosomal escape, and transfects the majority of cells in a large cell population.
										We demonstrate that NF55 mediates DNA delivery in vivo with gene induction efficiency that is comparable to commercial transfection reagents. In addition to gene induction in healthy mice, NF55/DNA nanoparticles showed promising tumor transfection in various mouse tumor models, including an intracranial glioblastoma model. The efficiency of NF55 to convey DNA specifically into tumor tissue increased even further after coupling a PEG2000 to the peptide via a disulphide-bond. Furthermore, a solid formulation of NF55/DNA displayed an excellent stability profile without additives or special storage conditions.
										Together, its high transfection efficacy and stability profile make NF55 an excellent vector for the delivery of DNA in vivo.">
										<br>
										<img SRC="NF55_MEP.png" width="35%" border="0" alt="">
										<img SRC="NF55_hphilphob.png" width="35%" border="0" alt="">
</center>
<br>
Freimann K., Arukuusk P., Kurrikoff K., Ferreira Vasconcelos L. D., Veiman K. -L., Uusna J., Margus H., <b>Garc&iacute;a-Sosa A. T.</b>, Pooga M., Langel &Uuml, "Optimization of in vivo DNA delivery with NickFect peptide vectors", <i>Journal of Controlled Release</i>, 2016, Vol. 241, 134-143. <a href="http://www.sciencedirect.com/science/article/pii/S0168365916307933">[HTML]</a>, <a href="GarciaSosa_JournalOfControlledRelease2016.pdf"> [DOI]</a>
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>

									<a name="pea16"><h3>Cell-penetrating peptides binding to siRNA</h3></a>
									The binding affinity of a series of cell-penetrating peptides (CPP) was modeled
									 through docking and making use of the number of intermolecular hydrogen bonds,
									  lipophilic contacts, and the number of sp3 molecular orbital hybridization carbons. 
									  The new ranking of the peptides is consistent with the experimentally determined efficiency in the downregulation of luciferase activity, which includes the peptides' ability to bind and deliver the siRNA into the cell. 
									  The predicted structures of the complexes of peptides to siRNA were stable throughout 10 ns long, explicit water molecular dynamics simulations. 
									  The stability and binding affinity of peptide-siRNA complexes was related to the sidechains and modifications of the CPPs, with the stearyl and quinoline groups improving affinity and stability. 
									  The reranking of the peptides docked to siRNA, together with explicit water molecular dynamics simulations, appears to be well suited to describe and predict the interaction of CPPs with siRNA.
									  <br>
									  <br>
<center>
<img SRC="Fig1_cpps.png" border="0" width="80%" alt="The binding affinity of a series of cell-penetrating peptides (CPP) was modeled through docking and making use of the number of intermolecular hydrogen bonds, lipophilic contacts, and the number of sp3 molecular orbital hybridization carbons. The new ranking of the peptides is consistent with the experimentally determined efficiency in the downregulation of luciferase activity, which includes the peptides' ability to bind and deliver the siRNA into the cell.
The predicted structures of the complexes of peptides to siRNA were stable throughout 10 ns long, explicit water molecular dynamics simulations. The stability and binding affinity of peptide-siRNA complexes was related to the sidechains and modifications of the CPPs, with the stearyl and quinoline groups improving affinity and stability. The reranking of the peptides docked to siRNA, together with explicit water molecular dynamics simulations, appears to be well suited to describe and predict the interaction of CPPs with siRNA.">
</center>
<br>

<br><b>Garc&iacute;a-Sosa A. T.,*</b> Tulp I., Langel K., Langel &Uuml;., "Peptide-Ligand Binding Modeling of siRNA with Cell-Penetrating Peptides",
									  <i>BioMed Research International</i>, 2014, Vol. 2014, Article ID 257040, 7 pages. DOI: <a href="http://dx.doi.org/10.1155/2014/257040">http://dx.doi.org/10.1155/2014/257040</a>, 
									  <a href="http://dx.doi.org/10.1155/2014/257040">[PDF]</a>.
									  <br>
									  <p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.<br></p>



<hr>
<a name="pea22"><h3>Mutations in MPK12 produce different activity in plant signalling</h3></a>
<center>
<img SRC="Fig1_MPK12_MPK12-G53R-600dpi.png" width="75%" border="0" alt="Plant gas exchange is regulated by guard cells that form stomatal pores. Stomatal adjustments are crucial for plant survival; they regulate uptake of CO2 for photosynthesis, loss of water, and entrance of air pollutants such as ozone. We mapped ozone hypersensitivity, more open stomata, and stomatal CO2-insensitivity phenotypes of the Arabidopsis thaliana accession Cvi-0 to a single amino acid substitution in MITOGEN-ACTIVATED PROTEIN (MAP) KINASE 12 (MPK12). In parallel, we showed that stomatal CO2-insensitivity phenotypes of a mutant cis (CO2-insensitive) were caused by a deletion of MPK12. Lack of MPK12 impaired bicarbonate-induced activation of S-type anion channels. We demonstrated that MPK12 interacted with the protein kinase HIGH LEAF TEMPERATURE 1 (HT1)—a central node in guard cell CO2 signaling—and that MPK12 functions as an inhibitor of HT1. These data provide a new function for plant MPKs as protein kinase inhibitors and suggest a mechanism through which guard cell CO2 signaling controls plant water management.">
<br>
<img SRC="Fig2_MPK4-MPK4-G55R-600dpi.png" width="75%" border="0" alt="">
<br>
<br>
</center>
Jakobson L., Vaahtera L., T&otilde;ldsepp K., Nuhkat M., Wang C., Wang Y.-S., H&otilde;rak H., Valk E., Pechter P., Sindarovska Y., Tang J., Xiao C., Xu Y., Talas U. G., <b>Garc&iacute;a-Sosa A. T.</b>, Kangasj&auml;rvi S., Maran U., Remm M., Roelfsema M. R. G., Hu H., Kangasj&auml;rvi J., Loog M., Schroeder J. I., Kollist H., Brosch&eacute; M., "Natural Variation in <i>Arabidopsis</i> Cvi-0 Accession Reveals an Important Role of MPK12 in Guard Cell CO<sub>2</sub> Signaling", <i>PLoS Biology</i>, 2016, Vol. 14, Iss. 12, e2000322, <a href="http://dx.doi.org/10.1371/journal.pbio.2000322">[DOI]</a>, <a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2000322">[HTML]</a>    
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>


<hr>
<a name="pea19"><h3>Small, low-toxicity, novel compounds that inhibit HIV-1</h3></a>
<center>
<img SRC="HIVrt_exp.jpg" width="50%" border="0" alt="A set of top-ranked compounds from a multi-objective in silico screen was experimentally tested for toxicity and the ability to inhibit the activity of HIV-1 reverse transcriptase (RT) in cell-free assay and in cell-based assay using HIV-1 based virus-like particles. Detailed analysis of a commercial sample that indicated specific inhibition of HIV-1 reverse transcription revealed that a minor component that was structurally similar to that of the main compound was responsible for the strongest inhibition. As a result, novel s-triazine derivatives were proposed, modelled, discovered, and synthesised, and their antiviral activity and cellular toxicity were tested. Compounds 18a and 18b were found to be efficient HIV-1 RT inhibitors, with an IC50 of 5.6 ± 1.1 μM and 0.16 ± 0.05 μM in a cell-based assay using infectious HIV-1, respectively. Compound 18b also had no detectable toxicity for different human cell lines. Their binding mode and interactions with the RT suggest that there was strong and adaptable binding in a tight (NNRTI) hydrophobic pocket. In summary, this iterative study produced structural clues and led to a group of non-toxic, novel compounds to inhibit HIV-RT with up to nanomolar potency">
</center>

Viira B., Selyutina A., <b>Garc&iacute;a-Sosa A. T.</b>, Karonen M., Sinkkonen J., Merits A., Maran U., "Design, Discovery, Modelling, Synthesis, and Biological Evaluation of Novel and Small, Low Toxicity s-Triazine Derivatives as HIV-1 Non-Nucleoside Reverse Transcriptase Inhibitors", <i>Bioorganic & Medicinal Chemistry</i>, 2016, Vol. 24, Iss. 11, 2519-2529. DOI: <a href="http://www.sciencedirect.com/science/article/pii/S0968089616302528">DOI</a>.
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.<br></p>

<a name="pea18"><h3>Virtual Screening for HIV-1 Protease</h3></a>
<center>
<img SRC="z36140438_n_dockedcogn_n_1bwb.png" width="25%" border="0" alt="A virtual screening to find novel inhibitors for HIV protease was performed on the ZINC database.1 A critical part in virtual screening and associated techniques is preliminary database filtering and size reduction and for that purpose a novel feature matrix matching procedure was used. The reduction of ∼14 million available ligands to a subset of 14299 ligands was achieved with a structure based approach where the analysis of the 3D structure of the active site of the protease produced a graph with hydrogen bond donor, hydrogen bond acceptor and hydrophobic subsites represented as graph nodes. A similar treatment was also applied to the compound database content and the comparison of binding site and ligand graphs was used to preselect potentially active ligands. The resulting set was further subjected to docking. The algorithm used was able to find several novel as well as previously known and experimentally tested ligands, demonstrating the validity of the approach.">
<br>
</center>
Takkis K., <b>Garc&iacute;a-Sosa A. T.</b>, Sild S. "Virtual Screening for HIV Protease Inhibitors Using a Novel Database Filtering Procedure", <i>Molecular Informatics</i>, 2015, Vol. 34, Iss. 6-7, 485-492. DOI: <a href="http://onlinelibrary.wiley.com/doi/10.1002/minf.201400170/full"> 10.1002/minf.201400170</a>, <a href="http://onlinelibrary.wiley.com/doi/10.1002/minf.201400170/abstract"> [Abstract]</a>, <a href="http://onlinelibrary.wiley.com/doi/10.1002/minf.201400170/epdf">[PDF]</a>.
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.<br></p>

<a name="pea17"><h3>Triangular numbers in re-ranking for HIV-1 Integrase</h3></a>
Ranking in virtual screening for HIV-1 integrase was improved using triangular numbers, which can group different runs from mutants, conformations and conditions. They are a growing series of numbers that in this work allow to define an objective threshold above which the majority of the top compounds are known inhibitors, while providing leeway for new compounds. Antitargets allow to profile the ligands according to five metabolizing and drug-effluxing enzymes and proteins.
<br>
<br>
<center>
<img SRC="Fig_triangular.gif" border="0" alt="A delicate balance exists between a drug molecule's toxicity and its activity. Indeed, efficacy, toxicity, and side effect problems are a common cause for the termination of drug candidate compounds and development projects. 
To address this, an antitarget interaction profile is built and combined with virtual screening and cross docking for new inhibitors of HIV-1 integrase, in order to consider possible off-target interactions as early as possible in a drug or hit discovery program. New ranking techniques using triangular numbers improve ranking information on the compounds and recovery of known inhibitors into the top compounds using different docking programs. This improved ranking arises from using consensus of ranks between docking programs and ligand efficiencies to derive a new rank, instead of using absolute score values, or average of ranks. The triangular number rerank also allowed the objective combination of results from several protein targets or screen conditions and several programs. 
Triangular number reranking conserves more information than other reranking methods such as average of scores or averages of ranks. In addition, the use of triangular numbers for reranking makes possible the use of thresholds with a justified leeway based on the number of available known inhibitors, so that the majority of the compounds above the threshold in ranks compare to the compounds that have known experimentally determined biological activity. 
The battery of anti- or off-targets can be tailored to specific molecular or drug design challenges. In silico filters can thus be deployed in successive stages, for prefiltering, activity profiling, and for further analysis and triaging of libraries of compounds.">
</center>
<br>

<br><b>Garc&iacute;a-Sosa A. T.,*</b> Maran, U., "Improving the Use of Ranking in Virtual Screening against HIV-1 Integrase with Triangular Numbers and Including Ligand Profiling with Antitargets", <i>Journal of Chemical Information and Modeling</i>, 2014, Vol. 54, Iss. 11, 3172-3185. DOI: <a href="http://dx.doi.org/10.1021/ci500300u">http://dx.doi.org/10.1021/ci500300u</a>,
<a href="http://pubs.acs.org/doi/pdf/10.1021/ci500300u">[PDF]</a>.
<br>
<a name="pea11"><h3>Anti-targets, ligand efficiency, and docking for HIV-1 RT, including some possible metabolic effects </h3></a>

We used ligand binding efficiencies in virtual screening, together with anti-targets composing several proteins involved in compound metabolism, in order to discover candidate compounds for wild-type and drug-resistant HIV-1 reverse transcriptases that have the best profiles, both for binding to a variety of targets (target proteins in several conformational, mutational, and hydration states), as well as their predicted interaction profile with PXR, SULT, and several CYPs.
<br><br>
<center>
<img SRC="HIVrt-F4-rev2.png" border="0" width="75%" alt="A.T. Garcia-Sosa, S. Sild, K. Takkis, U. Maran, Combined Approach using Ligand Efficiency, Cross-Docking, and Anti-Target Hits for Wild-Type and Drug-Resistant Y181C HIV-1 Reverse Transcriptase, Journal of Chemical Information and Modeling, J. Chem. Inf. Model. 2011, Vol. 51, Iss. 10, 2595-2611 docking binding mode x-ray crystal structure sulfotransferase SULT pregnane-x-receptor PXR cytochrome P450 2a6 2c9 3a4 CYP cross-docking cognate ligand inhibitor"><br></center>
<br>
Predicted positive and negative interaction profiles, both for pharmaceutical target inhibition and for the metabolic <i>in silico</i> profile of candidate compounds are thus developed. This may help in polypharmacology, systems pharmacology, drug repurposing, and other settings involving optimization to a variety of partners and chemical properties.
<br><br>
<center>
<img SRC="HIVrt-F5-rev.png" border="0" width="75%" alt="A.T. Garcia-Sosa, S. Sild, K. Takkis, U. Maran, Combined Approach using Ligand Efficiency, Cross-Docking, and Anti-Target Hits for Wild-Type and Drug-Resistant Y181C HIV-1 Reverse Transcriptase, Journal of Chemical Information and Modeling, J. Chem. Inf. Model. 2011, Vol. 51, Iss. 10, 2595-2611 metabolism side-effect toxicity liver intestines anti-targets score in silico docking binding mode x-ray crstal structure sulfotransferase SULT pregnane-x-receptro PXR cytochrome P450 2a6 2c9 3a4 CYP cross-docking cognate ligand inhibitor Abstract: New hits against HIV-1 wild-type and Y181C drug-resistant reverse transcriptases were predicted taking into account the possibility of some of the known metabolism interactions. 
In silico hits against a set of antitargets (<em>i.e.</em>, proteins or nucleic acids that are off-targets from the desired pharmaceutical target objective) are used to predict a simple, visual measure of possible interactions for the ligands, which helps to introduce early safety considerations into the design of compounds before lead optimization. 
This combined approach consists of consensus docking and scoring: cross-docking to a group of wild-type and drug-resistant mutant proteins, ligand efficiency (also called binding efficiency) indices as new ranking measures, pre- and postdocking filters, a set of antitargets and estimation, and minimization of atomic clashes. 
Diverse, small-molecule compounds with new chemistry (such as a triazine core with aromatic side chains) as well as known drugs for different applications (oxazepam, chlorthalidone) were highly ranked to the targets having binding interactions and functional group spatial arrangements similar to those of known inhibitors, while being moderate to low binders to the antitargets. 
The results are discussed on the basis of their relevance to medicinal and computational chemistry. 
Optimization of ligands to targets and off-targets or antitargets is foreseen to be critical for compounds directed at several simultaneous sites."
<br><br></center>
<br>
<b>Garc&iacute;a-Sosa A. T.,*</b> Sild S., Takkis K., Maran U., "Combined Approach using Ligand Efficiency, Cross-Docking, and Anti-Target Hits for Wild-Type and Drug-Resistant Y181C HIV-1 Reverse Transcriptase", <i>Journal of Chemical Information and Modeling</i>, 2011, Vol. 51, Iss. 10, 2595-2611. 
<br>
link: <a href="http://pubs.acs.org/doi/abs/10.1021/ci200203h">HTML</a>, <a href="http://pubs.acs.org/doi/pdf/10.1021/ci200203h">DOI</a>, <a href="ci200203h.pdf">[PDF]</a>. 
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.<br></p>
<hr>

<a name="pea15"><h3>Hydration of ligands in 2,332 crystal structures and their associated binding energies
</h3></a>
Recent findings are changing the view of water molecules in binding events, from a passive role to a fundamental and driving one. They can bridge protein-ligand interactions and guide their specificity, energetics, both enthalpically and entropically, and these outcomes depend on the shape, size, chemical, and dynamical nature of protein binding site and ligands.
High resolution X-ray crystal structures with reported experimentally determined K<i>i</i> or K<i>d</i> show that there is no statistically significant difference in binding energy between binding sites with tightly-bound water molecules and those without them. Other physicochemical properties and ligand efficiencies are also compared.
There are also benefits of lower log<i>P</i> and better
developability for tightly hydrated compounds, while stronger potency is not always required or beneficial. These results also hold for drugs/non-drugs comparisons.

<br>
<br>In addition, agonists and antagonists that use tightly bound water bridges are smaller, less lipophilic, and less planar;
have deeper ligand efficiency indices; and in general, possess better physicochemical properties for further development.
Therefore, tightly bound, bridging water molecules may in some cases be replaced and targeted as a strategy, though sometimes
keeping them as bridges may be better from a pharmacodynamic perspective.
The hydrated binding
site may be one of the many structure conformations available to the receptor, and different ligands will have a different ability to
select either tightly hydrated or non-tightly hydrated or non-hydrated receptor binding site conformations. 
<br>
<br>Compounds may thus be designed, and if a tightly
bound, bridging water molecule is observed in the binding site, attempts to replace it should only be made if the subsequent
ligand modification would improve also its ligand efficiency, enthalpy, specificity, and pharmacokinetic properties. If the
modification does succeed in replacing the tightly bound, bridging water molecule, it will have at least achieved benefits for ligand
optimization and development independently of either positive or negative change in binding affinity outcome.
<br>
<br>
<br>
<center>
<img SRC="TOC_watjcim2013.png" border="0" width="65%" alt="Some water molecules in binding sites are important for
intermolecular interactions and stability. The way binding site explicit
water molecules are dealt with affects the diversity and nature of
designed ligand chemical structures and properties. The strategies
commonly employed frequently assume that a gain in binding affinity
will be achieved by their targeting or neglect. However, in the present
work, 2332 high-resolution X-ray crystal structures of hydrated and
nonhydrated, drug and nondrug compounds in biomolecular
complexes with reported Ki or Kd show that compounds that use
tightly bound, bridging water molecules are as potent as those that do
not. The distribution of their energies, physicochemical properties, and
ligand efficiency indices were compared for statistical significance, and
the results were confirmed using 2000 permutation runs. Ligand cases
were also split into agonists and antagonists, and crystal structure pairs with differing tightly bound water molecules were also
compared. In addition, agonists and antagonists that use tightly bound water bridges are smaller, less lipophilic, and less planar;
have deeper ligand efficiency indices; and in general, possess better physicochemical properties for further development.
Therefore, tightly bound, bridging water molecules may in some cases be replaced and targeted as a strategy, though sometimes
keeping them as bridges may be better from a pharmacodynamic perspective. The results suggest general indications on tightly
hydrated and nontightly hydrated compounds in binding sites and practical considerations to adopt a strategy in drug and
molecular design when faced with this special type of water molecules. There are also benefits of lower log P and better
developability for tightly hydrated compounds, while stronger potency is not always required or beneficial. The hydrated binding
site may be one of the many structure conformations available to the receptor, and different ligands will have a different ability to
select either hydrated or nonhydrated receptor binding site conformations. Compounds may thus be designed, and if a tightly
bound, bridging water molecule is observed in the binding site, attempts to replace it should only be made if the subsequent
ligand modification would improve also its ligand efficiency, enthalpy, specificity, and pharmacokinetic properties. If the
modification does succeed in replacing the tightly bound, bridging water molecule, it will have at least achieved benefits for ligand
optimization and development independently of either positive or negative change in binding affinity outcome">
</center> 
<br><br><br>
<b>Garc&iacute;a-Sosa A. T.,*</b> 
"Hydration Properties of Ligands and Drugs in Protein Binding Sites:
Tightly-Bound, Bridging Water Molecules and Their Effects and
Consequences on Molecular Design Strategies"
, <i>Journal of Chemical Information and Modeling</i>, 2013, Vol. 53, Iss. 6, 1388-1405.
 DOI: <a href=
"http://pubs.acs.org/doi/pdf/10.1021/ci3005786">http://dx.doi.org/10.1021/ci3005786</a>, <a href="http://pubs.acs.org/doi/abs/10.1021/ci3005786"> [Abstract]</a>, <a href="http://pubs.acs.org/doi/pdf/10.1021/ci3005786">[PDF]</a>.

<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.<br></p>

<hr>
<a name="pea14"><h3>DrugLogit: Multivariate Logistic Regression, Drugs, Non-drugs, and Disease-Specificity</h3></a>
Simple and chemically intuitive logistic functions have been built to distinguish between drugs and non-drugs, taking into account individual disease categories or organ groups. 
These relate a few readily-available physicochemical molecular properties to a continuous and gradual outcome being the probability of classification as a drug, or drug-likeness.
<br>
<br>
<center>
<img SRC="TOCdruglogit.png" border="0" alt="Alfonso T. Garcia-Sosa, Mare Oja, Csaba Hetenyi, U. Maran, DrugLogit: Logistic Discrimination between Drugs and Nondrugs Including Disease-Specificity by Assigning Probabilities Based on Molecular Properties Journal of Chemical Information and Modeling, J. Chem. Inf. Model. 2012, Vol. 52, 2165-2180, ABSTRACT: The increasing knowledge of both structure and activity of compounds provides a good basis for enhancing the pharmacological characterization of chemical libraries. In addition, pharmacology can be seen as incorporating both advances from molecular biology as well as chemical sciences, with innovative insight provided from studying target-ligand data from a ligand molecular point of view. Predictions and profiling of libraries of drug candidates have previously focused mainly on certain cases of oral bioavailability. Inclusion of other administration routes and disease-specificity would improve the precision of drug profiling. In this work, recent data are extended, and a probability-based approach is introduced for quantitative and gradual classification of compounds into categories of drugs/nondrugs, as well as for disease- or organ-specificity. Using experimental data of over 1067 compounds and multivariate logistic regressions, the classification shows good performance in training and independent test cases. The regressions have high statistical significance in terms of the robustness of coefficients and 95% confidence intervals provided by a 1000-fold bootstrapping resampling. Besides their good predictive power, the classification functions remain chemically interpretable, containing only one to five variables in total, and the physicochemical terms involved can be easily calculated. The present approach is useful for an improved description and filtering of compound libraries. It can also be applied sequentially or in combinations of filters, as well as adapted to particular use cases. The scores and equations may be able to suggest possible routes for compound or library modification. The data is made available for reuse by others, and the equations are freely accessible at http://hermes.chem.ut.ee/~alfx/druglogit.html separation distinction">
<br><br></center>
<br>

See <a href="https://hermes.chem.ut.ee/~alfx/druglogit.html">Calculators Using These Equations</a>.
<br>
<a name="pea12"></a>
<br><b>Garc&iacute;a-Sosa A. T.,*</b> Oja M., Het&eacute;nyi C., Maran U., "DrugLogit: Logistic Discrimination between Drugs and Nondrugs Including Disease-Specificity by Assigning Probabilities Based on Molecular Properties", <i>Journal of Chemical Information and Modeling</i>, 2012, Vol. 52, Iss. 8, 2165-2180. DOI: <a href="http://pubs.acs.org/doi/pdf/10.1021/ci200587h">http://dx.doi.org/10.1021/ci200587h</a>, <a href="http://pubs.acs.org/doi/abs/10.1021/ci200587h"> [Abstract]</a>, <a href="druglogit.pdf">[PDF]</a>.
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.<br></p>
<h3>Description and characterization of drugs and non-drugs in chemical space through principal components analysis, and through probability density functions </h3>
<br>
<br>
<center><img SRC="F4abc-v4c-2column2-CMC-BW.png" width="80%" border="0" alt="A.T. Garcia-Sosa, C. Hetenyi, U. Maran, Molecular property filters describing pharmacokinetics and drug binding, Current Medicinal Chemistry, Curr. Med. Chem. journal 2012, Vol. 19, Iss. 11, 1646 - 1662 drug design drug likeness non-drugs Abstract: Drug-target binding affinity and pharmacokinetics are equally important factors of drug design.
Simple molecular properties such as molecular size have been used as pharmacokinetic and/or drug-likeness filters during chemical library design and also correlated with binding affinity.
In the present study, current property filters are reviewed, a collection of their optimal values is provided, and a statistical framework is introduced allowing calibration of their selectivity and sensitivity for drugs.
The role of ligand efficiency indices in drug design is also described.
It is concluded that the usefulness of property filters of molecular size and lipophilicity is limited as predictors of general drug-likeness.
However, they demonstrate increased performance in specific cases, <em>e.g.</em> in central nervous system diseases, emphasizing their future importance in specific, disease-focused library design instead of general drug-likeness filtering.
The physicochemical descriptor space has been extensively mapped and described in the literature for orally administered drugs and lead compounds. However, consideration of negative examples (non-drugs) or disease pathophysiology is not common in many studies. In the present work, a principal component analysis was carried out using drugs and non-drugs taking into account disease- and organ-specific categories, as well as different administration routes in addition to oral. The study involves 1386 relevant small-molecules including natural and synthetic products. Drug-specific as well as disease-category-specific or organ-specific regions and their respective threshold sets (ranges of descriptors) relative to non-drugs were elucidated on the scores plot and validated with external, independent sets of drugs and non-drugs. The respective loadings plot of molecular descriptors was rationalized in terms of physicochemically relevant groups related to the components of solvation free energy. The results of this analysis can contribute to the improved profiling of drug candidates and libraries making use of disease- and organ-specificity coded by physicochemical descriptors and ligand binding efficiency."><br><br></center>

<b>Garc&iacute;a-Sosa A. T.</b>, Maran U., Het&eacute;nyi C., "Molecular property filters describing pharmacokinetics and drug binding", <em>Current Medicinal Chemistry</em>, 2012, Vol. 19, 1646-1662.
<br>
<a href="Garcia-SosaATcurrmedchem2012forweb.pdf"> [PubMed 22376034]</a>,
<a href="Garcia-SosaATcurrmedchem2012forweb.pdf"> [DOI]</a>, 
<a href="Garcia-SosaATcurrmedchem2012forweb.pdf"> [HTML]</a>, 
<a href="Garcia-SosaATcurrmedchem2012forweb.pdf"> [PDF]</a>.
<br><br>

<b>Garc&iacute;a-Sosa A. T.,*</b> Oja M., Het&eacute;nyi C., Maran U., "Disease-specific differentiation between drugs and non-drugs using principal component analysis of their molecular descriptor space", <i>Molecular Informatics</i>, 2012, Vol. 31, 369-383. 
<br>
<a href="http://onlinelibrary.wiley.com/doi/10.1002/minf.201100094/pdf"> [DOI]</a>,
<a href="http://onlinelibrary.wiley.com/doi/10.1002/minf.201100094/abstract"> [HTML]</a>,
<a href="Garcia-SosaATmolinf2012PCA.pdf"> [PDF]</a>.
<br>
<br>
<b>Garc&iacute;a-Sosa A. T.,*</b> and Maran U., "Drugs, non-drugs, and disease category specificity: Organ effects by ligand pharmacology", <i>SQER</i>, 2013, Vol. 24, 585-597.
<br>
<a href="http://dx.doi.org/10.1080/1062936X.2013.773373"> [DOI]</a>,
<a href="http://www.tandfonline.com/doi/full/10.1080/1062936X.2013.773373"> [HTML]</a>,
<a href="Garcia-SosaAT_sqer2013-3.pdf"> [PDF]</a>.
<br>

<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.<br></p>

<hr>
<br>
<b>2011: <a href="http://dmmsclick.wiley.com/view.asp?m=5hqmcmtcm7wylvqocueu&u=2351173&f=h">Named 'Hot Article in Biochemistry' by Wiley</a></b>
 
<a name="pea10"><h3>Chemically Relevant Functional Group Substitutions of a Tightly-Bound Water Molecule Including Enthalpic, Entropic, Fully Explicit Solvent and Ground State Effects</h3></a>

New, modified ligands for the Abl tyrosine kinase-SH3 domain implicated in chronic myelogenous leukemia (CML, a cancer of white blood cells) were built by generating different scenarios: including, neglecting, and targeting a specific hydration site with several chemical functional groups. This enabled determining the thermodynamic and structural effects of these chemical probes in a protein-ligand complex that has a tightly-bound water molecule bridging the interaction. Molecular dynamics using thermodynamic integration (a relative of the free energy perturbation method), explicit water, periodic boundary conditions, full thermodynamic cycles to include the desolvation effects of ligands as well as of the individual water molecule, and correcting for the ground state provides valuable information on the different groups that are best suited chemically and physically for substitution. Some water molecules are loosely bound to the protein or ligand surface, and so will not interfere or interact strongly. However, there are some which will be strongly bound and can be distinguished in several complexed states of the protein (see WaterScore below). This paper shows the different routes available for modification of the ligand structure based on a strongly-bound water, and their effects and insight into ligand optimization of protein&ndash;ligand&ndash;water systems.
<br>
<br>
<br>
<center>

<img SRC="figstates.jpg" border="0" width="60%" alt="A.T. Garcia-Sosa, R.L. Mancera, Free Energy Calculations of Mutations Involving a Tightly Bound Water Molecule and Ligand Substitutions in a Ligand&ndash;Protein Complex, Molecular Informatics, Mol Inf, 2010, Vol. 29, Iss. 8-9, 589-600 Abl-kinase SH3 domain CML chronic myelogenous leukemia myeloid alchemical states free energy water themodynamic integration"><br><br></center>
<br><br> 
The energy of extracting a water molecule from the bulk solvent of water molecules has the reverse sign, but the same magnitude, as the energy of replacing a water molecule into the bulk solvent.
<br><br><br>
<center>
<img SRC="hydwat.png" border="0" alt="A.T. Garcia-Sosa, R.L. Mancera, Free Energy Calculations of Mutations Involving a Tightly Bound Water Molecule and Ligand Substitutions in a Ligand-Protein Complex, Molecular Informatics, Mol Inf, 2010, Vol. 29, Iss. 8-9, 589-600 cancer tumor inhibitor water molecule free energy solvation hydration drug design docking molecular dynamics Abstract: The accurate calculation of the free energy of interaction of protein&ndash;water&ndash;ligand systems has an important role in molecular recognition and drug design that is often not fully considered. 
We report free energy thermodynamic integration calculations used to evaluate the effects of inclusion, neglect, and targeting and removal (<em>i.e.</em>, systematic substitution by ligand functional groups) of an important, tightly bound, water molecule in the SH3 domain of Abl tyrosine kinase. 
The effects of this water molecule on the free energies of interaction of several Abl-SH3 domain-ligand systems reveal that there is an unfavourable free energy change associated with its removal into the bulk solvent. 
Only three substitutions by an additional functional group (out of methyl, ethyl, hydroxyl, amino, and amide groups) in the phenyl ring of a tyrosine in the peptide ligand resulted in a favourable change in the free energy of binding upon replacement of the ordered water molecule. 
This computational approach provides a direct route to the systematic and rigorous prediction of the thermodynamic influence of ordered, structural water molecules on ligand modification and optimization in drug design by calculating free energy changes in protein&ndash;water&ndash;ligand systems.
"><br><br></center>
<br>
<br><b>Garc&iacute;a-Sosa A. T.*</b> and Mancera R.L., "Free Energy Calculations of Mutations Involving a Tightly Bound Water Molecule and Ligand Substitutions in a Ligand-Protein Complex", <em>Molecular Informatics</em>, 2010, Vol. 29, Iss. 8-9, 589-600.
<br>
link: <a href="http://onlinelibrary.wiley.com/doi/10.1002/minf.201000007/full">HTML</a>,
<a href="Garcia-SosaAT_molinf2010.pdf">DOI</a>,
<a href="Garcia-SosaAT_molinf2010.pdf"> [PDF]</a>.
<br><br>
(typo: p 594, l 14 should read "atomic fluctuation = 0.89 &Aring, effective k = 2 kcal / mol / &Aring ") <br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.<br></p>

<hr>

<a name="pea9"><h3>Ligand and drug efficiency indices (EI)</h3></a>

Efficiency indices effectively normalize the free energy of binding of a drug or ligand
per a given measure of that compound's molecular characteristics, such as molecular weight or number of heavy atoms.
 Particular ligand efficiencies were determined to improve the correlation between experimental and calculated EI values for protein-drug complexes.
 &Delta;G/W (free energy of binding divided by Wiener index, a topological measure), &Delta;G/NoC (free energy of binding divided by number of carbons, a measure of size and lipophilicity), and &Delta;G/P (free energy of binding divided by the octanol / water partition coefficent, a measure of lipophilicity and hydrophilicity) produced improved correlations for EI for several docking programs and scoring functions. 
Better correlations between experimental and calculated EI values can improve the accuracy of virtual screening and molecular docking.
 In addition, the common bias of scoring functions in favor of larger ligands can be removed.  
<br><br><br>
<center>

<img SRC="Figjcc_ei.jpg" border="0" width="75%" alt="A.T. Garcia-Sosa, C. Hetenyi, U. Maran, Drug Efficiency Indices for Improvement of Molecular Docking Scoring Functions, Journal of Computational Chemistry, J. Comput. Chem. 2010, Vol. 31, Number 1, 174-184 ligand efficiencies, binding efficiencies, drug Abstract:
A dataset of protein-drug complexes with experimental binding energy and crystal structure were analyzed and the performance of different docking engines and scoring functions (as well as components of these) for predicting the free energy of binding and several ligand efficiency indices were compared. 
The aim was not to evaluate the best docking method, but to determine the effect of different efficiency indices on the experimental and predicted free energy. 
Some ligand efficiency indices, such as &Delta;G/W (Wiener index), &Delta;G/NoC (number of carbons), and &Delta;G/P (partition coefficient), improve the correlation between experimental and calculated values. 
This effect was shown to be valid across the different scoring functions and docking programs. 
It also removes the common bias of scoring functions in favor of larger ligands. Compound collection filtering can be done prior or after docking, using pharmacokinetic as well as pharmacodynamic profiles. 
Achieving these better correlations with experiment can improve the ability of docking scoring functions to predict active molecules in virtual screening.
"><br><br></center>
<br>


<br><b>Garc&iacute;a-Sosa A. T.,*</b>  Het&eacute;nyi C., Maran U., "Drug Efficiency Indices for Improvement of Molecular Docking Scoring Functions", <em>Journal of Computational Chemistry</em>, 2010, Vol. 31, Number 1, 174-184.
<br> 
link: <a href="GarciaSosaAT2010jcc.pdf">DOI </a>
,
<a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/122374448/HTMLSTART
"> HTML</a>, 
<a href="GarciaSosaAT2010jcc.pdf"> [PDF]</a>, 
<a href="GarciaSosaAT2010jcc.pdf">[authorPDF]</a>.
<br><br>
<b>Garc&iacute;a-Sosa A. T.,*</b> Sild S., Maran U.,"Docking and Virtual Screening
 Using Distributed Grid Technology", <em>QSAR & Combinatorial Science</em>, 2009,
 Vol. 28, Number 8, 815-821.
<br>
link: <a href="Garcia-SosaAT09QSAR.pdf">DOI </a>
,
<a href="http://www3.interscience.wiley.com/journal/122462308/abstract?CRETRY=1&SRETRY=0"> HTML</a>, 
<a href="Garcia-SosaAT09QSAR.pdf">[authorPDF]</a>.

<br>
 
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>

<hr>

<a name="pea8"><h3>Inhibitor design targeting H5N1 avian influenza</h3></a>

Drug design was conducted together with virtual screening for generating inhibitors that
simultaneously occupy several binding sites of both wild-type
H5N1 avian influenza neuraminidase, as well as of a drug-resistant mutant.
Several ligand efficiency (also called binding efficiency) values were
used to better characterize the compounds.
<br><br><br>

<center>
<img SRC="FigH5N1jcim.png" border="0" alt="A.T. Garcia-Sosa, S. Sild, U. Maran, Design of Multi-Binding-Site Inhibitors, Ligand Efficiency, and Consensus Screening of Avian Influenza H5N1 Wild-Type Neuraminidase and of the Oseltamivir-Resistant H274Y Variant, Journal of Chemical Information and Modeling, J. Chem. Inf. Model. 2008, Vol. 48, 2074-2080 H5N1 influenza neuraminidase docking virtual screening The binding sites of wild-type avian influenza A H5N1 neuraminidase, as well as those of the Tamiflu (oseltamivir)-resistant H274Y variant, were explored computationally to design inhibitors that target simultaneously several adjacent binding sites of the open conformation of the virus protein. 
The compounds with the best computed free energies of binding, in agreement by two docking methods, consensus scoring, and ligand efficiency values, suggest that mimicking a polysaccharide, &beta;-lactam, and other structures, including known drugs, could be routes for multibinding site inhibitor design. 
This new virtual screening method based on consensus scoring and ligand efficiency indices is introduced, which allows the combination of pharmacodynamic and pharmacokinetic properties into unique measures.
"><br><br></center>
<br>
<b><a href="https://hermes.chem.ut.ee/~alfx/videos.html">See Animation</a></b>.
<br>
<br><b>Garc&iacute;a-Sosa A. T.,*</b> Sild S., Maran U., "Design of Multi-Binding-Site Inhibitors, Ligand Efficiency, and Consensus Screening of Avian Influenza H5N1 Wild-Type Neuraminidase and of the Oseltamivir-Resistant H274Y Variant", <em>Journal of Chemical Information and Modeling</em>, 2008, Vol. 48, 2074-2080. 
<br>
link: 
<a href="http://dx.doi.org/10.1021/ci800242z"> DOI</a> ,
<a href="http://pubs3.acs.org/acs/journals/supporting_information.page?in_manuscript=ci800242z"> Supporting Information</a>   
, 
<a href="Garcia-SosaAT08.pdf">[PDF]</a>. 
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>
<hr>

<a name="pea7"><h3>Drug design with metalloproteins, zinc centres and multiple molecular dynamics simulations</h3></a>

Small molecule inhibitors of Botulinum neurotoxin
 serotype A (BoNTA) were designed using multiple molecular dynamics simulations (MMDS)
  and a non-bonded four coordination active site zinc model,
   as well as projects on protein-ligand free energy of binding and other
    thermodynamic analyses of multiple protein-ligand systems.
    <br><br>
    The use of 20 molecular dynamics simulations
    of at least 2 ns in explicit solvent
    results in
     improved sampling that provides a good measure to discriminate if a ligand
     in a given binding mode has favourable interactions inside a binding site,
     with the best
      ligands remaining in a strong coordination to the protein and having favourable interaction energy terms.

<br><br>
<center>

<img SRC="figforwebpcs4.png" border="0" alt="J.G. Park, P.C. Sill, E.F. Makiyi, A.T. Garcia-Sosa, C.B. Millard, J.J. Schmidt, Y.P. Pang, Serotype-selective, Small-molecule Inhibitors of the Zinc Endopeptidase of Botulinum Neurotoxin Serotype A, Bioorganic & Medicinal Chemistry, Bioorg. Med. Chem., 2006, Vol. 14, 395-408 Tetrahedral zinc protein ligand inhibitor drug design molecular dynamics Botulinum neurotoxin serotype A (BoNTA) is one of the most toxic substances known. 
Currently, there is no antidote to BoNTA. 
Small molecules identified from high-throughput screening reportedly inhibit the endopeptidase &ndash; the zinc-bound, catalytic domain of BoNTA &ndash; at a drug concentration of 20 &micro;M. 
However, optimization of these inhibitors is hampered by challenges including the computational evaluation of the ability of a zinc ligand to compete for coordination with nearby residues in the active site of BoNTA. 
No improved inhibitor of the endopeptidase has been reported. 
This article reports the development of a serotype-selective, small-molecule inhibitor of BoNTA with a K<sub><em>i</em></sub> of 12 &micro;M. 
This inhibitor was designed to coordinate the zinc ion embedded in the active site of the enzyme for affinity and to interact with a species-specific residue in the active site for selectivity. 
It is the most potent small-molecule inhibitor of BoNTA reported to date. 
The results suggest that multiple molecular dynamics simulations using the cationic dummy atom approach are useful to structure-based design of zinc protease inhibitors.
"><br><br></center>
<br><br>
This page has a movie that shows part of an MD simulation of the 12 micromolar 
inhibitor of BoNTA.
The zinc atom center is shown as a sphere in magenta:
<br>
<b><a href="https://hermes.chem.ut.ee/~alfx/videos.html">See Movie</a>.</b>

      <br><br>
      Park J.G., Sill P.C., Makiyi E.F., <b>Garcia-Sosa A. T.</b>, Millard C.B., Schmidt J.J., Pang Y.P., "Serotype-selective, Small-molecule Inhibitors of the Zinc Endopeptidase of Botulinum Neurotoxin Serotype A", <em>Bioorganic & Medicinal Chemistry</em>, 2006, Vol. 14, 395-408.
<br> link:
<a href="http://dx.doi.org/10.1016/j.bmc.2005.08.018">DOI</a>
, <a href="BoNTA_inh.pdf">[PDF]</a>. 
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>
<hr>

<a name="pea6"><h3>Explicit, crystallographic water molecules in computer-aided drug design</h3></a>

Hydration of protein-substrate/inhibitor 
complexes is a factor involved in the design of ligands. 
Water molecules have a very important role in the 
molecular recognition process as well as in the overall thermodynamics of
protein-ligand binding and 
biomolecular processes. Therefore, dynamic simulation studies of proteins 
in an aqueous environment are critical for a realistic description of their 
structure and thermodynamic properties. 
A study of both the structural and dynamical properties of these systems 
proves useful to understand ligand&ndash;binding phenomena.<br><br>

A first step in
these studies was a comprehensive survey of patterns in
hydration sites as seen in the crystal structures of sets of proteins of interest.

A multivariate function
was developed to fit the properties of the water molecules observed both in the X
ray crystal structures of the active sites of <em>apo</em> 
proteins and in the structure of the
actives sites of complexes of the same proteins, to distinguish them from  
the properties of those water molecules not matched between sites. 
These comparisons were carried out for a variety of proteins. 
The predicted outcome variable (called WaterScore value) may be used to assign 
the probability of observing a particular water molecule from the <em>apo</em> 
protein structure in a structure of a complex of that protein, provided that 
it is not displaced sterically by the ligand or by a protein conformational 
change. The group of water molecules used in the analysis are those in the 
hydration shell of the ligand in the complex structure of the protein. 
This is interesting for drug design, docking strategies and methods
because it allows the inclusion of a set of explicit water molecules, since
the bound water molecules can be scored very easily and fast. 
The inclusion of specific water molecules allows for a better description of protein-ligand binding and improves the calculation of binding poses, intermolecular interactions and binding energy. 

<br><br><br><center>
<img SRC="Figure1-1.jpg" border="0" alt="A.T. Garcia-Sosa, R.L. Mancera, P.M. Dean, WaterScore: A Novel Method for Distinguishing between Bound and Displaceable Water Molecules in the Crystal Structure of the Binding Site of Protein-Ligand Complexes, Journal of Molecular Modeling, J. Mol. Model., 2003, Vol. 9, Issue 3, 172-182 hydration sites water molecules X-ray crystal structure protein ligand drug Abstract: We have performed a multivariate logistic regression analysis to establish a statistical correlation between the structural properties of water molecules in
the binding site of a free protein crystal structure, with the probability of observing the water molecules in the same location in the crystal structure of the ligand&ndash;complexed form. 
The temperature B-factor, the solvent-contact surface area, the total hydrogen bond energy, and the number of protein-water contacts were found to discriminate between bound and displaceable water molecules in the best regression functions obtained. 
These functions may be used to identify those bound water molecules that should be included in structure-based drug design and ligand docking algorithms.
"><br><br></center>
<br>
<align="justified">
This figure shows the binding site (in thin
sticks) of penicillopepsin (protein databank code 3app.pdb) 
with its crystallographically-determined
water molecules (in spheres) and superimposed ligand (in thick
sticks, from structure 1ppk.pdb). Water molecules sterically replaced by
the ligand upon complexation are shown in cyan. It is interesting to see that
many atoms of the ligand are on sites previously occupied by water molecules
in the <em>apo</em> protein crystal structure, and/or are using the contacts to
the protein that these waters made. Bound water molecules (those found in both
structures) are shown in blue. Displaced water molecules (those appearing in
only the <em>apo</em> structure and not clashing with the ligand upon binding)
are shown in yellow.
Water molecules removed from the analysis due to a lack of hydrogen bonds
with the protein are shown in white.

<a name="pea5"></a>
<br><br>
<br>
You can use the WaterScore Calculator<sup>&copy;</sup> below to type in the values of the properties of water molecules and compute their WaterScore value. 
SCSA is the solvent accessible contact surface area (in &Aring;<sup><small>2</small></sup>), 
and NPAC is the number of protein atomic contacts the water molecule has within a distance of 3.5 &Aring;.
<br><br<br>
<br<br>

<center>
<h3><u>WaterScore Calculator <sup>&copy;</sup></u></h3>
<br>
<form name="calcer">

Bfactor <input type="text" name="inputA" value="0" size=7><br>
SCSA <input type="text" name="inputB" value="0" size=7><br>
NPAC <input type="text" name="inputC" value="0" size=7><br><br>


<input type="button" value="Calculate" onClick="calcIt()">

<p>_______</p>

WaterScore value:&nbsp;&nbsp;&nbsp;

<input type="text" name="output" size=32><br>

</form>

<br><br><u>Note:</u> A value of NaN indicates a very small value, close to zero. 
You need to have JavaScript enabled on your web browser. 
As a rough guide, a completely exposed water molecule would have a surface area of approx. 24.63 &Aring;<sup>2</sup>.<br><br>

</center>

If you use or modify the function, please cite:<br>
<br>
<b>Garc&iacute;a-Sosa A. T.,*</b> Mancera R. L., Dean P. M. "WaterScore: A Novel Method for Distinguishing between Bound and Displaceable Water Molecules in the Crystal Structure of the Binding Site of Protein-Ligand Complexes", <em>Journal of Molecular Modeling</em>, 2003, Vol. 9, Issue 3, 172-182.

<small>
<br>
(copyright Springer-Verlag 2003).
The original publication is available at this link: 
<a href="http://link.springer.de/link/service/journals/00894/contents/03/00129/" border=0>metapress</a> , 
or by digital object identifier 
</small>
<a href="http://dx.doi.org/10.1007/s00894-003-0129-x" border=0>DOI</a>, 
<a href=waterscore.pdf>[PDF]</a>. 
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>

<hr>
<a name="pea4"><h3>Tight waters in pharmacophores</h3></a>

Tightly-bound water molecules were also found to to be neccessary to explain 
pharmacophore binding model projection points. This showed that active ligands used
tightly bound water molecules as interaction groups inside protein binding sites. 
<br><br><br><center>
<img SRC="pharmforwat.png" border="0" alt="D.G. Lloyd, A.T. Garcia-Sosa, I.L. Alberts, N.P. Todorov, R.L. Mancera, The Effect of Tightly Bound Water Molecules on the Structural Interpretation of Ligand-Derived Pharmacophore Models, Journal of Computer-Aided Molecular Design, J. Comput.-Aid. Mol. Design, 2004, Vol. 18, 89-100 pharmacophore ligand protein drug water molecules alignment Abstract: The importance of the consideration of water molecules in the structural interpretation of ligand-derived pharmacophore models is explored. 
We compare and combine results from recently introduced methods for bound-water molecule identification in protein binding sites and ligand-superposition-based pharmacophore derivation, for the interpretation of ligand-derived pharmacophore models. 
In the analysis of thymidine kinase (HSV-1) and poly (ADP-ribose) polymerase (PARP), the concurrent application of both methods leads to an agreement in the prediction of tightly bound water molecules as key pharmacophoric points in the binding site of these proteins. 
This agreement has implications for approaching binding site analysis and consensus drug design, as it highlights how pharmacophore-based models of binding sites can include interaction features not only with protein groups but also with bound water molecules.
"><br><br></center>
<br>
<align="justified">

Lloyd D.G., <b>Garc&iacute;a-Sosa A. T.</b>, Alberts I.L., Todorov N.P., Mancera R.L. "The Effect of Tightly Bound Water Molecules on the Structural Interpretation
of Ligand-Derived Pharmacophore Models", <em>Journal of Computer-Aided Molecular Design</em>, 2004, Vol. 18, 89-100.
<br>
link:
<a href="http://www.springerlink.com/link.asp?id=r7855l7004480347">
metapress</a>, or   

<a href="http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1023/B:jcam.0000030032.81753.b4">
DOI
</a>
, <a href="waterphore.pdf">[PDF]</a>. 
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>

<hr>
<a name="pea3"><h3>PARP inhibitor design</h3></a>
Tightly bound water molecules have also been studied for their effects
in ligand docking and de novo structure based drug design for several protein target cases.
<br>
For example, poly(ADP-Ribose) polymerase, PARP: <br><br>
<br>
<center>
<img SRC="tableparpfig.png" border="0" alt="A.T. Garcia-Sosa, S. Firth-Clark, R.L. Mancera, Including Tightly-Bound Water Molecules in De Novo Drug Design. Exemplification Through the In Silico Generation of Poly (ADP-Ribose) Polymerase Ligands, Journal of Chemical Information and Modeling, J. Chem. Inf. Model. 2005, Vol. 45, 624-633, drug design PARP poly ADP-ribose polymerase ligand drug discovery water Abstract: Different strategies for the in silico generation of ligand molecules in the binding site of poly(ADP-ribose)polymerase (PARP) were studied in order to observe the effect of the targeting and displacement of tightly bound water molecules. 
Several molecular scaffolds were identified as having better interactions in the binding site when targeting one or two tightly bound water molecules in the NAD binding site. 
Energy calculations were conducted in order to assess the ligand&ndash;protein and ligand&ndash;water&ndash;protein interactions of different functional groups of the generated ligands. 
These calculations were used to evaluate the energetic consequences of the presence of tightly bound water molecules and to identify those that contribute favorably to the binding of ligands.
" ><br><br>
</center>
where in addition to the generation of ligands we also analyzed some functional groups on ligands to identify those best suited 
to displace a tightly-bound water molecule in the binding site. 
The amount of energy neglected by not considering tightly-bound water molecules
bridging the protein-ligand interaction was also analyzed.
<br><br>

<b>Garc&iacute;a-Sosa A. T.,*</b> Firth-Clark S., Mancera R. L. "Including Tightly-Bound Water Molecules in De Novo Drug Design. Exemplification Through the In Silico Generation of Poly (ADP-Ribose) Polymerase Ligands", <em>Journal of Chemical Information and Modeling</em>, 2005, Vol. 45, 624-633. 
<br>

<a href="http://dx.doi.org/10.1021/ci049694b">DOI</a>

, 
<a href="http://pubs.acs.org/doi/full/10.1021/ci049694b"> [HTML]</a>, 
<a href="parp.pdf">[PDF]</a>. 
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>

<hr>

<a name="pea2"><h3>CDK2 inhibitor design</h3></a>

In the case of Cyclin dependent kinase, CDK2:<br>

<center>
<img SRC="webfigcdk2wat.jpg" border="0" width="45%" alt="A.T. Garcia-Sosa, R.L. Mancera, The Effect of a Tightly-Bound Water Molecule on Scaffold Diversity in the Computer-Aided de novo Ligand Design of CDK2 Inhibitors, Journal of Molecular Modeling, J. Mol. Model. 2006, Vol. 12, Issue 4, 422-431 CDK2 water drug design de novo discovery ligand Abstract: We have determined the effects that tightly bound water molecules have on the de novo design of cyclin-dependent kinase-2 (CDK2) ligands. 
In particular, we have analyzed the impact of a specific structural water molecule on the chemical diversity and binding mode of ligands generated through a de novo structure-based ligand generation method in the binding site of CDK2. 
The tightly bound water molecule modifies the size and shape of the binding site and we have found that it also imposed constraints on the observed binding modes of the generated ligands. 
This in turn had the indirect effect of reducing the chemical diversity of the underlying molecular scaffolds that were able to bind to the enzyme satisfactorily.
"><br></center>


a tightly-bound water molecule was observed to modulate the chemical diversity 
of the ligands generated.

<align="justified">

<br><br>
<b>Garc&iacute;a-Sosa A. T.*</b> and Mancera R. L. "The Effect of a Tightly-Bound Water Molecule on Scaffold Diversity in the Computer-Aided <em>de novo </em> Ligand Design of CDK2 Inhibitors", <em>Journal of Molecular Modeling</em>, 2006, Vol. 12, Issue 4, 422-431.
<small>
<br>
The original publication is available at this link: 
   <a href="http://www.springerlink.com/link.asp?id=758856x82rm19250">metapress</a>
   , or by digital object identifier:
</small>
   <a href="http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-005-0063-1">DOI</a>
,   
<a href="gal_cdk2.pdf">[PDF]</a>. 
<br>
<p style="font-size:85%;">See <a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a>.</p>

<hr>

<a name="pea1"><h3><em>Ab initio</em> studies of Iron-Oxygen systems, Fe<sub>n</sub>O<sub>n</sub><sup>+,-</sup></h3></a>

During my undergraduate studies, I did research on transition metal-ligand
systems with ab initio methods such as DFT.
<br><br><br><br>
<center>
<img SRC="webFeO2aa.png" border="0" alt="Iron oxygen DFT orbital">       
<img SRC="webfigFeO2bb.jpg" border="0" alt="A.T. Garcia-Sosa, M. Castro, A Density Functional Study of FeO2, FeO2+, and FeO2-, International Journal of Quantum Chemistry, 2000, Vol. 80, Issue 3, 307-319 Iron oxygen DFT vibration frecuency frequencies Abstract The lowest energy structures of FeO<sub>2</sub> were determined by means of density functional theory techniques as implemented in the program DGauss 3.0.1.
The calculations performed were of the all-electron type using two levels of theory, namely the local spin density approximation with the use of the Vosko&ndash;Wilk&ndash;Nusair (VWN) functional and the generalized gradient approximation (GGA) in the form of the Becke (1988) exchange and Perdew (1986) correlation functionals.
Results were visualized with UniChem.
Bond distances and angles as well as total energies were calculated for several states of the moieties: Fe(O)<sub>2</sub>, <em>C</em><sub>2<em>v</em></sub>; Fe(O)<sub>2</sub>, <em>D</em><sub>&infin;<em>h</em></sub>; Fe(&eta;<sup>2</sup>-O<sub>2</sub>), <em>C</em><sub>2<em>v</em></sub>; Fe(&eta;<sup>1</sup>-O<sub>2</sub>), <em>C</em><sub><em>s</em></sub>; and Fe(&eta;<sup>1</sup>-O<sub>2</sub>), <em>C</em><sub>&infin;<em>v</em></sub>.
Molecular orbital and harmonic vibrational analyses were carried out for these species, in addition to Mulliken population analyses.
Singly positive and negative charged species were also considered and fully geometry optimized in a self-consistent field (SCF) gradient method.
Accurate ionization potentials and electron affinities (both vertical, <em>v</em>, and adiabatic, <em>a</em>, determinations) were thus able to be computed.
The results show the following for the ground state (GS) Fe(O)<sub>2</sub>, <em>C</em><sub>2<em>v</em></sub>, <em>M</em>=3: &ang;OFeO=138.1&deg; (133.6&deg;) [values in parentheses are for local spin density approximation (LSDA)&ndash;VWN, while the others are at the GGA-B88/P86 level].
Here, <em>R</em><sub><em>e</em></sub> Fe&ndash;O=1.60 &Aring; (1.57 &Aring;), ET=&minus;1414.2064 au (&minus;1,410.5047 au), EA<sub><em>a</em></sub>=2.47 (2.60) eV, IP<sub><em>a</em></sub>=10.6 (10.5) eV, EA<sub><em>v</em></sub>=2.41 (2.20) eV, IP<sub><em>v</em></sub>=10.67 (10.63) eV, and EA<sub>exp</sub>=2.349 eV (in agreement with related studies).
In the GS the dioxygen molecule is found to be dissociated, compared to those states which have coordination modes where the O<sub>2</sub> molecule formally persists.
A 3<em>d</em>4<em>sp</em> configuration for the iron atom is found to be especially relevant in Fe&ndash;O bond formation.
The iron&ndash;oxygen and oxygen&ndash;oxygen bonds involved are characterized.
A direct relationship is observed between these electronic and structural properties, influencing also the total energy for a given molecule.
"><br><br>
</center> 
<align="justified">

<br> 
<b>Garc&iacute;a-Sosa A. T.</b> and Castro M., "A Density Functional Study of FeO<sub>2</sub>, FeO<sub>2</sub><sup>+</sup>, and FeO<sub>2</sub><sup>- </sup>", <em>International Journal of Quantum Chemistry</em>, 2000, Vol. 80, Issue 3, 307-319. 


<br>
link: 
<a href="http://www3.interscience.wiley.com/cgi-bin/abstract/73501022/START" border=0>
HTML
</a> , or <a href="http://dx.doi.org/10.1002/1097-461X(2000)80:3<307::AID-QUA4>3.0.CO;2-8">
DOI </a>
, <a href="ijqc_paper.pdf">[PDF]</a>
, <a href="feo2gtex.txt">[TEX]</a>.

</align>
<br>

</div>
<br>
</align>
<hr>
<h3><a href="https://hermes.chem.ut.ee/~alfx/publications.html">Publications</a></h3>
<h3><a href="https://hermes.chem.ut.ee/~alfx/videos.html">Videos</a></h3>
<h3><a href="https://hermes.chem.ut.ee/~alfx/inthenews.html">In The News</a><h3>
<h3><a href="https://github.com/AlfonsoTGarcia-Sosa/ML">Garcia-Sosa Github</a></h3>
<h3><a href="https://hermes.chem.ut.ee/~alfx/research.html#pea6">WaterScore online calculator</a></h3> 
<h3><a href="https://hermes.chem.ut.ee/~alfx/druglogit.html">DrugLogit online calculator</a></h3>
<h3><a href="https://hermes.chem.ut.ee/~alfx/FEP_Charge_Corrections.html">FEP Charge Corrections</a></h3>

<h3><a href="https://hermes.chem.ut.ee/~alfx/download.html">Download</a></h3>
<h3><a href="https://hermes.chem.ut.ee/~alfx/index.html">Contact</a></h3>
<br><br><br>


<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-7383406-1");
pageTracker._trackPageview();
} catch(err) {}</script>

</body>
</html>






